### PR TITLE
Update ChunkSection.cpp

### PR DESCRIPTION
**Pull request by IridescentRose**
Fixes drawing issues.
screen shot 2017-10-11 at 13 45 50

### CODE REVIEW

**Comment by Hopson97** [comment]
AHH of course ! xD

Thank you!

**Comment by IridescentRose** [comment]
You may wish to change the Flora Shader since it puts vertices in the ground (inVert.y -= 0.2;)  and change the motion to something more like [this](https://github.com/minetest/minetest/blob/master/client/shaders/nodes_shader/opengl_vertex.glsl). Because water and leaves don't move the same XD

**Comment by IridescentRose** [comment]
And I mean only the part that pertains to the leaves, not the water, since yours looks better than theirs.

**Comment by Hopson97** [comment]
Yes, I already planned for leaves shader :)

Thanks again

File: Source/World/Chunk/ChunkSection.cpp [hunk 1]
```diff
@@ -81,6 +81,7 @@ void ChunkSection::bufferMesh()
 {
     m_meshes.solidMesh.bufferMesh();
     m_meshes.waterMesh.bufferMesh();
+    m_meshes.waterMesh.bufferMesh();
```
Comment (qwertychouskie) [inline Source/World/Chunk/ChunkSection.cpp:84]: This is repeated twice?

Comment (Hopson97) [reply Source/World/Chunk/ChunkSection.cpp:84]: I fixed this before merging into master :)

**Pull request merged**